core/src/Kokkos_Array.hpp

```diff
@@ -195,8 +195,10 @@ struct Array<T, 0> {
     return *reinterpret_cast<const_pointer>(-1);
   }
 
-  KOKKOS_INLINE_FUNCTION pointer data() { return nullptr; }
-  KOKKOS_INLINE_FUNCTION const_pointer data() const { return nullptr; }
+  KOKKOS_INLINE_FUNCTION constexpr pointer data() { return nullptr; }
```

Comment:
```suggestion
  KOKKOS_FUNCTION constexpr pointer data() { return nullptr; }
```

(you already get `inline` implicitly)

Reply:
I'd agree with you but here consistency win and I really don't want to touch all function members.